contrib/sandbox/packages/st2-mistral/actions/transformer.py

```diff
@@ -0,0 +1,60 @@
+import json
+import os
+import six
+import yaml
+
+from st2actions.runners import pythonrunner
+from st2actions.runners.mistral import utils
+from st2client import client
+
+
+class MistralDSLTransformer(pythonrunner.Action):
```

Comment:
I mean that's what he does. 

from st2actions.runners.mistral import utils

But I agree this could have been a separate PR. I was very confused too.